zammad/Chart.yaml

```diff
@@ -17,17 +17,17 @@ maintainers:
 dependencies:
   - name: elasticsearch
     repository: https://helm.elastic.co
-    version: 7.16.2
+    version: 7.16.3
     condition: zammadConfig.elasticsearch.enabled
   - name: memcached
-    version: 5.15.5
+    version: 6.0.2
```

Comment:
As this is a major version upgrade only raising the bugfix chart version seems a bit risky.
The braking changes of the memcached chart should be mentioned in the upgrading section of this chart too.